docs/source/getting_started/SettingsEditor.rst

```diff
@@ -0,0 +1,809 @@
+     a file.
+   - Browse - use the controls to navigate to find config.json
+   - Search - set the starting directory to search and open the
+     **Search** drop-down
+   - When you see the file click on it to select it and
+     click **Select File** button (below the file browser). The **Select File**
+     button will act on the last file you selected - whether in the file directory
+     listing or the search results.
+   - optionally, click **View Settings** to confirm that the settings in the
+     ``config.json`` look right
+
+3. Convert the config settings to msticpyconfig format
+
+   - click **Convert to MP**
+   - click **View Settings**
+
+4. Save the file
+
+   - type a path into the **Current file** text box
+   - Click on **Save file**
+
+5. You can set this file to always load by assigning the path to
+   an environment variable. See `Setting the path to your
+   msticpyconfig.yaml <#Setting-the-path-to-your-msticpyconfig.yaml>`__
+
+.. code:: ipython3
+
+    mpconfig = MpConfigFile()
+    mpconfig
+
+.. figure:: _static/settings_mp_file_config_view.png
+   :alt: Imported settings from config.json
+
+This is the equivalent of the previous steps from the command line, which
+is probably faster if you know where your config file is.
+
+.. code:: ipython3
+
+    mpconfig = MpConfigFile()
+    mpconfig.load_from_file("E:\\src\\asi-jupyter\\notebooks\\config.json")
+    mpconfig.map_json_to_mp_ws()
+    mpconfig.settings
+
+
+.. parsed-literal::
+
+    {'AzureSentinel': {'Workspaces': {'ASWorkspace': {'ResourceGroup': 'ASWorkspaceRG',
+        'SubscriptionId': '2c1bc08d-6a9e-43f0-aa55-a84f383b1a6b',
+        'TenantId': '3f3fb3df-3c79-4407-8043-4a765430db0e',
+        'WorkspaceId': '722f602f-340e-47a1-8d0a-c2b6a8fbc4c4'}}}}
+
+
+
+Edit your msticpyconfig settings
+--------------------------------
+
+MpConfigEdit
+~~~~~~~~~~~~
+
+To do this we'll be using the configuration editor MpConfigEdit.
+It uses a tabbed interface to show different settings sections.
+
+.. figure:: _static/settings_mp_conf_edit.png
+   :alt: Configuration editor Interface.
+
+As well as showing settings, there is text box to input a file name to
+save the settings to - with a corresponding **Save File** button.
+
+.. note:: Each settings tab has a **Save** button. This saves changes on
+   the current editor screen to an in-memory copy of your settings. They
+   are not saved to a file until you enter a file name and click on
+   **Save File**.
+
+The **Validate Settings** also lets you check on the status of the settings.
+It does a logical check (for sections/settings that you probably need) as
+well as a structural check to make sure that setting syntax is specified
+correctly. The results of the validation are shown in a text view box.
+Close this when you are done with the **Close** button.
+
+There is also help for the settings available on each tab, with links
+to more detailed documenation.
```

Comment:
documentation

Reply:
Deleted - a forgotten experiment

Reply:
I missed the typos - will put them into a current branch